blog/2024-03-21-scio-case-study.md

```diff
@@ -0,0 +1,101 @@
+
+<br></br>
+
+## Translating CSO to Spanish & French
+
+The Climate Security Research team at Alliance Bioversity & CIAT includes native Spanish and French speakers, who carefully did all the translations of the platform. If any text was missed during the translation, the platform developers used **machine translations as a backup**. Then, other skilled native speakers in the Climate Security Research team checked these machine translations to make sure they were accurate.
+
+![nigeria page in french](/img/blog/scio-case-study/nigeria_french.webp)
+
+<br></br>
+
+> _“Having a **central reference tool** where multiple users can join and translate content is a major shift from the [old-school approach of sharing spreadsheet files](/blog/context-gamechanger-localization#dont-use-spreadsheets-for-software-localization) where mistakes are difficult to avoid.“_
+
+<br></br>
+
+After translating the platform, they observed a **significant uptick in platform visitors from Spanish and French-speaking countries** compared to the initial version, which was exclusively available in English. Specifically, there was a notable 30% surge in both the number of visits and the average duration spent on the platform.
+
+:::tipSCiO doesn't stop only with the CSO Platform
+
+SCiO is currently supporting the **Ethiopian Digital AgroClimate Advisory Platform** [(EDACaP)](https://edacap.ethioagroclimate.net/#/Home), aiming to translate it from English to Ethiopian language using Tolgee. In collaboration with the Ethiopian Ministry of Agriculture and local research centers, it serves as a **decision support system for farmers, policymakers, and researchers**, leveraging historical data to optimize yield and production.
+
+:::
+
+## Localization Best Practice Revealed
+
+As we wrapped up, we asked Panagiotis one last question: If you had to choose one, what would you say is the most important software localization best practice?
+
+_A clear process that allows translators to provide their translations in a **CI/CD philosophy**. Organizing the translations in stages and connecting them with the actual CI/CD pipeline of a product leads to quicker and error-free deployments._
+
+## Tolgee for Open-Source
+
+In Tolgee, as fans of open-source, we would love to support the **open-source ecosystem, so our Business plan is free for all open-source projects like CSO**. To get this offer, reach out to us by [email](mailto:info@tolgee.io) or by DM in our [Slack community](https://Tolg.ee/slack).
```

Comment:
Weird formulation:
In Tolgee, as fans of open-source, we would love to ...

Maybe
As an open-source fans, we would love to ... (Tolgee is already in the title)